tests/compiletest.rs

```diff
@@ -21,12 +21,11 @@ fn run_mode(mode: &'static str) {
     }
 
     let mut config = compiletest::default_config();
-    let cfg_mode = mode.parse().ok().expect("Invalid mode");
 
-    config.mode = cfg_mode;
+    config.mode = mode.parse().ok().expect("Invalid mode");
     config.src_base = PathBuf::from(format!("tests/{}", mode));
     let dir = out_dir.to_str().unwrap();
-    config.target_rustcflags = Some(format!("-L {} -L {}/deps", dir, dir));
+    config.target_rustcflags = Some(format!("-L {}/build -L {}/deps", dir, dir));
```

Comment:
Huh, hadn't seen that dir before but it looks like it just contains the output of the build script (or have it changed in a recent nightly?).